.gitignore

```diff
@@ -17,3 +17,8 @@ doc_src/auto_api/
 doc_src/auto_examples/
 auto_figures/
 enhancement.png
+pyspecProcScripts/B12TMPS0004_1.5.8.bin
+pyspecProcScripts/bossac.exe
+pyspecProcScripts/getBoardInfo.bat
+pyspecProcScripts/uploadFirmware.bat
+"\357\200\272q"
```

Comment:
rather than adding to gitignore, just delete these changes.

Reply:
I know how to delete the files themselves.... but how do I delete the changes?

Reply:
Just delete the files, and then go into .gitignore, and remove these lines.